generators/app/templates/jsconfig.json

```diff
@@ -2,13 +2,8 @@
   "compilerOptions": {
     "baseUrl": "./src",
     "paths": {
-      "utils/*": [
-        "utils/*"
-      ]
+      "utils/*": ["utils/*"]
     }
   },
-  "exclude": [
-    "node_modules",
-    "dist"
-  ]
-}
\ No newline at end of file
+  "exclude": ["node_modules", "dist"]
```

Comment:
I believe this was shifted originally by prettier, I am wondering if the v2 update is what is now catching this or was it you manually?

Reply:
I did everything manually. `generate --> commit --> re-generate` and everything that shows up as changed should probably be changed in templates too.